source/community/release_notes/quince.rst

```diff
@@ -15,10 +13,19 @@ These are the release notes for the Quince release, the 17th community release o
 Breaking Changes
 ****************
 
+* `Include and enable the Indigo theme in the default Open edX image <https://github.com/overhangio/tutor/issues/953>`__
+  * Deployers running OpenEdX with the default theme and no additional plugins will have the Indigo theme enabled automatically.
+* `Deployers must ensure that for all of their IDAs, any JWT_PUBLIC_SIGNING_JWK_SET Django setting does not contain whitespace inside of the Base64 strings of the encoded keys <https://github.com/openedx/edx-drf-extensions/blob/master/CHANGELOG.rst#880---2023-05-16>`__
+* Django-storages upgraded to latest version and it has some breaking changes.
+  * The constructor kwarg bucket is no longer accepted. Instead, use bucket_name.
+  * define default_acl value explicitly in constructor kwarg e.g default_acl: public-read. in previous versions django-storages provides default value as public-read but now it is none. So it's important to mention it explicitly as per your use case.
+* For Django 4.2 CSRF_TRUSTED_ORIGINS must include scheme. Update all urls there with schemes. e.g .edx.org changes to ``https://*.edx.org``.
```

Comment:
Could you please add that the minimum Docker requirements have changed? They are now: Docker v24.0.5+, with BuildKit 0.11+ https://github.com/overhangio/tutor/pull/920/files#diff-379774f5c6e537f2001c951863c6ac577b56eb5c6e61cc581dce95143760ff36

Reply:
Added to breaking changes. Thanks for the report!